vendor/src/github.com/docker/libnetwork/driverapi/driverapi.go

```diff
@@ -58,11 +58,17 @@ type InterfaceInfo interface {
 	// The API is to be used to assign both the IPv4 and IPv6 address types.
 	SetIPAddress(ip *net.IPNet) error
 
+	// Assign extra IP addresses
+	SetIPAddresses(ips []*net.IPNet) error
```

Comment:
Rename to SetExtraIPAddresses() to be "consistent" with the getter? (The same way SetIPAddress() is "consistent" with Address(); not sure why they left out "IP" in the getter...)